packages/perseus/src/widgets/radio/widget.jsx

```diff
@@ -419,16 +450,17 @@ class Radio extends React.Component<Props> {
             const numCorrect = props.numCorrect;
 
             for (let i = 0; i < choicesSelected.length; i++) {
-                // $FlowFixMe[prop-missing]
-                const index = props.choices[i].originalIndex;
-                // $FlowFixMe[incompatible-use]
-                choicesSelected[index] = choiceStates[i].selected;
+                if (props.choices[i].originalIndex != null) {
```

Comment:
**question:** What was preventing this code from running before?  It might be good to add a comment here that this check is solely to make Flow happy.

Reply:
On line 454, originalIndex is accessed from props.choices[i]. Because originalIndex is optional, it was giving an error (see img). I will add a comment for sure.
<img width="864" alt="image" src="https://user-images.githubusercontent.com/60857422/198049740-e19713b7-df5e-4a2f-a925-cbf5a216532c.png">

Reply:
I was able to remove the null check in #314.